package/itest/ubuntu.sh

```diff
@@ -29,7 +29,8 @@ export TZ=US/Pacific
 ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
 apt-get update && apt-get install -y software-properties-common
-apt-get install -y --force-yes python3.7 python3-pip python3-yaml awscli
+add-apt-repository ppa:deadsnakes/ppa
```

Comment:
!8ball this should be gated in an `if YELP` conditional?

Reply:
Hmmm, this seems fine to me to just use anyways, and I imagine quite a few other builds do the same